fabric-tool-attributes-v1/build.gradle

```diff
@@ -0,0 +1,8 @@
+archivesBaseName = "fabric-tool-attributes-v1"
+version = getSubprojectVersion(project, "0.1.0")
+
+dependencies {
+    compile project(path: ':fabric-api-base', configuration: 'dev')
+    compile project(path: ':fabric-tag-extensions-v0', configuration: 'dev')
+    compileOnly 'com.google.code.findbugs:jsr305:3.0.2'
```

Comment:
Afaik this breaks java 9 and above. Consider using something else like checker qual or jetbrain annotations.

Reply:
Oh boy, it does seem like JSR305 got busted in JDK9. I could maybe use Spotbugs, which is a continuation of Findbugs. I'm not sure how much I like the concept of using IDE-specific annotations.

Reply:
Moved to Spotbugs.